src/notebooks/controllers/kernelSource/remoteNotebookKernelSourceSelector.ts

```diff
@@ -213,7 +226,32 @@ export class RemoteNotebookKernelSourceSelector implements IRemoteNotebookKernel
                 },
                 onDidChangeItems: onDidChangeItems.event
             });
+            if (provider.extensionId === JVSC_EXTENSION_ID) {
+                quickPick.onDidChangeValue(async (e) => {
+                    if (!provider.getQuickPickEntryItems) {
+                        return;
+                    }
+                    const quickPickCommandItems = [];
+                    if (quickPickServerItems.length > 0) {
+                        quickPickCommandItems.push({
+                            label: Common.labelForQuickPickSeparatorIndicatingThereIsAnotherGroupOfMoreItems,
+                            kind: QuickPickItemKind.Separator
+                        });
+                    }
 
+                    const commands = await provider.getQuickPickEntryItems(e);
+                    const newProviderItems: KernelProviderItemsQuickPickItem[] = commands.map((i) => {
```

Comment:
is there something special about this group that can be described better than 'more'?

Reply:
(I'm not really sure what it currently looks like)

Reply:
This is existing code. More iss Jessy a separator, blue line

Reply:
Will demo next week